2021q3/doceng.adoc

```diff
@@ -0,0 +1,28 @@
+////
+Quarter:	3rd quarter of 2021
+Prepared by:	dbaio
+Reviewed by:	gjb, carlavilla
+Last edit:	$Date: 2021-10-03 05:34:14 -0300 (Sun, 03 Oct 2021) $
+Version:	$Id: doceng-2021-3rd-quarter-status-report.adoc 185 2021-10-03 08:34:14Z carlavilla $
+////
+
+=== Documentation Engineering Team
+
+Link: link:https://www.freebsd.org/docproj/[FreeBSD Documentation Project] +
+Link: link:https://docs.freebsd.org/en/books/fdp-primer/[FreeBSD Documentation Project Primer for New Contributors] +
+Link: link:https://www.freebsd.org/administration/#t-doceng[Documentation Engineering Team]
+
+Contact: FreeBSD Doceng Team <doceng@FreeBSD.org>
+
+The doceng@ team is a body to handle some of the meta-project issues associated with the FreeBSD Documentation Project, more infomation on link:https://www.freebsd.org/internal/doceng/[FreeBSD Doceng Team Charter].
```

Comment:
s/Project, more infomation on/Project; for more information, see/ (mind the missing "r" in "information" even if you don't agree with the whole suggestion)

Reply:
Thanks. I've just pushed a fix as suggested.